README.md

```diff
@@ -7,3 +7,9 @@
 - `npm install`
 - `bower install`
 - `grunt build`
+
+## New Version of Stache
```

Comment:
Should this new content be above the existing content? Or maybe replace it altogether? Seems like to me, but maybe it's just me.

Reply:
I wasn't sure if I should replace it or not. What do you think?

Reply:
My personal opinion is that we should replace it. I don't feel strongly about it, but replacing that content makes it clear that we don't want people using Stache 1. Leaving the existing content feels like it leaves the door open just a little, which is all some people need.

Reply:
√ Done, old content removed.